apps/ledger-live-mobile/src/newArch/features/WalletSync/__integrations__/shared.tsx

```diff
@@ -41,3 +42,24 @@ export function WalletSyncSharedNavigator() {
     </QueryClientProvider>
   );
 }
+
+export const INITIAL_TEST = (state: State) => ({
+  ...state,
+  settings: {
+    ...state.settings,
+    readOnlyModeEnabled: false,
+    overriddenFeatureFlags: {
+      llmWalletSync: {
+        enabled: true,
+        params: {
+          environment: "STAGING",
+          watchConfig: {
```

Comment:
We can set the `watchConfig` to an empty object here as it will then take the default values